test/succeed/non-conjectures/non-conjectures.test

```diff
@@ -106,7 +107,7 @@ Interface
   current = Foo
   context = TopCtx
   modules
-    scope 
+    scope
```

Comment:
The `make non_conjectures` command is failing due to this change. Revert it please.